demo/CMakeLists.txt

```diff
@@ -17,7 +17,6 @@ set(PROTO_COMPILED_SRCS "")
 # Loop through each proto and compile using protoc
 foreach(PROTO
   content_analysis/sdk/analysis
-  content_analysis/sdk/handshake
   content_analysis/sdk/acknowledgement
```

Comment:
Why remove handshake but not acknowledgement?  Both just defined one enum and it seems it would be easier to just define those enums in analysis proto.